src/lib/rest/rest.cpp

```diff
@@ -518,6 +518,18 @@ static char* removeTrailingSlash(std::string path)
 */
 int servicePathSplit(ConnectionInfo* ciP)
 {
+  //
+  // Special case: empty service-path 
+  //
+  if ((ciP->httpHeaders.servicePathReceived == true) && (ciP->servicePath == ""))
```

Comment:
This is related with the last comments in https://github.com/telefonicaid/fiware-orion/issues/843?